serverless/serverless-release-notes.adoc

```diff
@@ -23,6 +23,14 @@ include::modules/serverless-deprecated-removed-features.adoc[leveloffset=+1]
 
 // Release notes included, most to least recent
 // OCP + OSD + ROSA
+include::modules/serverless-rn-1-25-0.adoc[leveloffset=+1]
+// 1.25.0 additional resources, OCP docs
+ifdef::openshift-enterprise[]
+[role="_additional-resources"]
+.Additional resources
+* xref:../serverless/security/serverless-config-tls.adoc#serverless-config-tls[Configuring TLS authentication]
```

Comment:
This cross-reference produces a "Page Not Found" error in the preview and I can't find the target page in the `openshift-enterprise` distro at that location.